Include in Parser a Review Manager

digoreis opened the pull request:
I include Review Manager in Parser and Proposal file. I prepared to recipe multiple Reviewers like Authors. 

Also, include a reviewBy in the proposal to help filter. 

Closes #1

Commit:
Include in Parser a Review Manager

Include filter by Reviewer

Commit:
Travis fix

jessesquires commented on Sources/Parser.swift:
```diff
@@ -83,14 +88,26 @@ func proposalFrom(fileContents: String, fileName: String) -> Proposal {
-
+    
+    let reviewManagers = reviewManagerFromString(reviewManagerLine)
+    dump(Proposal(title: title,
```
nit: remove?

jessesquires commented on Sources/Parser.swift:
```diff
@@ -83,14 +88,26 @@ func proposalFrom(fileContents: String, fileName: String) -> Proposal {
                     fileContents: fileContents,
-                    wordCount: words)
+                    wordCount: words
+                    )
```
let's put `)` on the same line to follow existing project style 😄

jessesquires commented on Sources/Proposal.swift:
```diff
@@ -88,6 +95,17 @@ extension Proposal {
+        return reviewManagers.contains(reviewer)
+    }
+    
+    public func reviewBy(_ reviewerName: String) -> Bool {
```
let's make these `reviewedBy` 😄

jessesquires commented on Sources/ReviewManager.swift:
```diff
@@ -0,0 +1,49 @@
+//  License
+//  Copyright © 2016 Jesse Squires
+//  Released under an MIT license: http://opensource.org/licenses/MIT
+//
```
let's copy the project comment headers

jessesquires reviewed (changes requested):
Thanks so much @digoreis ! 🎉 

Overall this looks great, just a few minor things to address.

Also -- could we revert the proposal/submodule changes? That will make this cleaner for git history.

Commit:
Remove log, rename method, fix headers

Commit:
Remove proposal from Github

Commit:
Merge with last version in master

digoreis commented:
Hi @jessesquires, I updated the pull request. To fix the proposal in git history I remove of the project and include de directory in the gitignore file. But the better solution is to remove a copy in the update, creating a symbolic link to the swift-proposal directory. 

And I fixed the others comments in Code Review. Sorry the dump, I forget to remove the dump 😢.

jessesquires commented on .gitignore:
```diff
@@ -64,3 +64,4 @@ fastlane/screenshots
 # https://github.com/johnno1962/injectionforxcode
 
 iOSInjectionProject/
+swift-proposal-analyzer.playground/Resources/proposals/
```
we don't want to do this, or delete the proposals... 😅

jessesquires commented:
Thanks @digoreis !

Looks like more unrelated changes were added. We definitely don't want to change the `gitignore`.

Do you want to re-pull the repo, and do a fresh PR with only the changes to the framework source files? 

Just *don't* run the `update_proposals.sh` script and it should be ok.

jessesquires commented:
Hey @digoreis - did you want open up a new PR for this? 😄
